warehouse/forklift/legacy.py

```diff
@@ -869,7 +869,12 @@ def file_upload(request):
             if field.description and isinstance(field, wtforms.StringField):
                 error_message = (
                     "{value!r} is an invalid value for {field}. ".format(
-                        value=field.data, field=field.description
+                        value=(
+                            field.data[:60] + "..."
```

Comment:
```suggestion
                            field.data[:30] + "..." + field.data[-30:]
```
Just to give a sense of what the entire string is.

Reply:
it'd be a little awkward for strings just over the limit... since we would return 63 chars for a 61 char field.. but makes sense :-D